app/Providers/AppServiceProvider.php

```diff
@@ -66,5 +67,24 @@ public function boot()
                 Settings::get('timezone')
             )->utc();
         });
+
+        Command::macro('projectArgument', function () {
+            /** @var Command $this */
+            $project = $this->argument('project');
+
+            if (! $project) {
+                $project = getenv('default_project');
+            }
+
+            if (! $project) {
+                $project = Settings::get('default_project');
+            }
+
+            return $project;
+        });
+
+        if (file_exists($localConfig = getcwd().DIRECTORY_SEPARATOR.'.hours')) {
+            Dotenv::create(getcwd(), '.hours')->load();
```

Comment:
This should probably be moved to an event listener. The `AppServiceProvider` is re-ran for every test case (or any other time the app boots) but we only want to load the dotenv file when the console application boots.

[Here's an example of registering a listener](https://github.com/matt-allan/hours/blob/1edba464d637e75e0709e1998fe56d7ca7a9f56f/app/Providers/EventServiceProvider.php#L20) and [here is an example listener](https://github.com/matt-allan/hours/blob/1edba464d637e75e0709e1998fe56d7ca7a9f56f/app/Events/InitializeDatabase.php).

Reply:
Good catch, updated the PR to reflect this change